example/linux/main.cc

```diff
@@ -0,0 +1,6 @@
+#include "my_application.h"
+
+int main(int argc, char** argv) {
```

Comment:
What is 'example/linux/' for?

Reply:
To debug the example app as a linux desktop app. It has a better debugger.